.github/workflows/ci.yaml

```diff
@@ -5,9 +5,11 @@ on: [ push ]
 jobs:
   ci:
     runs-on: [ ubuntu-latest ]
+    env:
+      GOMARKDOC_OPTS: -c
     strategy:
       matrix:
-        target: [ test, lint ]
+        target: [ test, lint, docs ]
```

Comment:
Does this simply generate the documentation or actually verify that the generated content is identical to what's in the repository?

Reply:
Oh, the `GOMARKDOC_OPTS=-c` does that?

I guess it's fine but could be more discoverable to still have a `verify-docs` target and set the environment there. I don't feel strongly about this, though.